### PR TITLE
Add typescript typings

### PR DESCRIPTION
**Pull request by LucianBuzzo**

### CODE REVIEW

**Comment by LucianBuzzo** [comment]
@10xjs Can this PR be merged? It'd be really useful for us Typescript users!

**Comment by LucianBuzzo** [comment]
@izaakschroeder Can this PR be merged?

**Review by izaakschroeder — COMMENTED** [review]
One thought, otherwise LGTM! Can merge after that's tidied up I think 🎉 Thanks for the addition @LucianBuzzo 😄

File: typings/react-resize-observer.d.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    width: number;
+    height: number;
+    top: number;
+    right: number;
+    bottom: number;
+    left: number;
+  }
+
+  export interface ResizeObserverProps {
+    onResize?: (react: DOMRect) => any;
```
[the remark this answers is not in the document]
Comment (LucianBuzzo) [reply typings/react-resize-observer.d.ts]: ah yes! Good spot :)

**Comment by LucianBuzzo** [comment]
@izaakschroeder Thanks for the review, I've updated the arguments names - should be good to go now!

**Comment by LucianBuzzo** [comment]
@izaakschroeder Can you take another look at this please?

**Comment by izaakschroeder** [comment]
LGTM! 🎉

**Comment by izaakschroeder** [comment]
All set as part of `1.1.1` @LucianBuzzo 😄

**Pull request merged**